spec/factories/achiever/course_delegate.rb

```diff
@@ -0,0 +1,24 @@
+FactoryBot.define do
+  factory :course_delegate, class: "Achiever::Course::Delegate" do
+    course_occurence_no { Faker::Number.number(digits: 8) }
+    course_template_no { Faker::Number.number(digits: 8) }
+    is_fully_attended { "false" }
+    online_cpd { false }
+    progress { "157420003" }
```

Comment:
Not essential, but we could add some traits for the different progress states, just to make it easier to test in the future?